main.py

```diff
@@ -1,14 +1,14 @@
 import pymongo
 from pymongo import MongoClient
 from fastapi import FastAPI, Request
-from pydantic.main import BaseModel
-from pydantic import root_validator, ValidationError
-from enum import Enum
-from datetime import date
+from src.models import ProductCreate
+from src.settings import MONGO_URI, MONGO_DB
 
-app = FastAPI(
-    title="product"
-)
+
+app = FastAPI()
+
+client = MongoClient(MONGO_URI)
+mongo_db = client[MONGO_DB]
```

Comment:
These variables should be created under startup event and attached to app's state. Something like

```python
app.state = MongoClient(...)
```